asf_heat_pump_suitability/pipeline/prepare_features/grid_capacity.py

```diff
@@ -0,0 +1,758 @@
+import re
+import os
+import shutil
+from typing import Union, Any
+
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+from shapely.geometry.base import BaseGeometry
+from shapely import wkb
+
+from asf_heat_pump_suitability.getters.s3_getters import (
+    load_s3_data,
+    get_shapefile_from_s3,
+)
+from asf_heat_pump_suitability.getters import get_datasets
+
+# Constants
+CRS = "EPSG:4326"  # Geometry coordinate reference system - standard longitude/latitude projection
+TAKEUP_RATE = 30  # Percentage of households in each LSOA to install heat pumps
+POWER_PER_HEATPUMP = 8  # Power rating per heat pump in kW
+COEFFICIENT_OF_PERFORMANCE = (
+    2.5  # Heat output (heat pump rating) / electricity consumed
+)
+SUBSTATION_SCALING_FACTOR = 1.0  # Factor to scale substation power rating (MVA) by
```

Comment:
Would be great to get ASF team colleagues inputs on these constants to check they're appropriate